development/Writing-a-plugin.md

```diff
@@ -8,18 +8,18 @@ There are 3 important steps when you want to create and implement a new chat ove
 
 ## Create a new plugin project
 
-Use the *Create Plugin* configuration or start the custom task `sbt create` directly. Enter the basic plugin information in the command promt. This includes the name, version and base plugin folder (e.g. `plugins-public`). The command creates the folder structure of the new plugin and adds a basic build file. Here you can also add custom dependencies.
+Use the *Create Plugin* run configuration or start the custom task `sbt create` directly. Enter the basic plugin information in the command promt. This includes the name, version and base plugin folder (e.g. `plugins-public`). The command creates the folder structure of the new plugin and adds a basic build file. Here you can also add custom dependencies.
 
-Next, run the custom task `sbt fetch` to let the framework look for new plugins and update the plugin reference file `plugins.sbt`. If you're using IntelliJ, you can now reload the framework by hand to register the plugin as new and custom project. You should also run `sbt reload` (Yes, both actions have different effects although they shouldn't).
+Next, run the custom task `sbt fetch` to let the framework look for new plugins and update the plugin reference file `plugins.sbt`. If you're using IntelliJ, you can now reload the framework by hand to register the plugin as new as well as a custom project. You should also run `sbt reload` (Yes, both actions have different effects although they shouldn't).
```

Comment:
s/by hand/manually
`register the plugin as new as well as a custom project` just sound horrible, what about "register the plugin as a new custom project"